CONTRIBUTING.md

```diff
@@ -19,6 +19,74 @@ The Valkey project is led by a Technical Steering Committee, whose responsibilit
 * Want to help with documentation? [Move on to valkey-doc](https://github.com/valkey-io/valkey-doc)
 * Report a vulnerability? See [SECURITY.md](SECURITY.md)
 
+## How to Install clang-format 18 on a Debian Box
```

Comment:
It would also be good to put this under a code format section, instead of a section for installing clang format. It would be nice to include an example for how to run it.